Connected System Plug-in (CSP) Examples/Major Versioned Template/.idea/gradle.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
do we usually at the .idea files?

Reply:
Yeah, I looked at the other CSPs and we commit everything 🤷